azure/eventprocessorhost/abstract_event_processor.py

```diff
@@ -13,12 +13,14 @@ def __init__(self, params=None):
         pass
 
     @abstractmethod
-    async def open_async(self, context):
+    async def open_async(self, context, event_processor_state=None):
```

Comment:
Why is this value called `event_processor_state` when it's called `event_processor_context` elsewhere?

Reply:
Fixed.